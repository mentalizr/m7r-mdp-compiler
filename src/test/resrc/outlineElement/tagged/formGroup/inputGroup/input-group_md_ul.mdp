@input-group[id="0815-2"]
    * erster
    * zweiter
    * dritter

    ### Eine Ãœberschrift

    Hier noch ein Praragraph

Hier ein weiterer Abschnitt