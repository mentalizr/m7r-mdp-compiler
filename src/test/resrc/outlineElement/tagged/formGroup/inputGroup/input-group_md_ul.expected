<div class="form-group">
    <ul class="mb-2">
        <li>erster</li>
        <li>zweiter</li>
        <li>dritter</li>
    </ul>
    <p class="h3 mt-4 mb-4">Eine Ãœberschrift</p>
    <p>Hier noch ein Praragraph</p>
    <input type="text" class="form-control ns_input" id="0815-2" placeholder="">
</div>